include/control.h

```diff
@@ -52,6 +53,7 @@ class Config {
 public:
 	std::vector<std::string> startup_params = {};
 	std::vector<std::string> configfiles = {};
+	std::set<std::string> loadedConfigFIles = {};
```

Comment:
On variable names:
 - `configfiles`
 - `loadedConfigFIles`

Does the `configfiles` hold config files that aren't loaded?

Maybe renaming them both to differentiate their meaning:
 -  config_files_from_user
 -  config_files_canonical_paths

(or something more terse!)

Reply:
`configfiles` is an array that's used elsewhere in the codebase (`mount` or `imgmount` & `config -l`), so I don't really want to change its name.

Original implementation was to use that vector for detecting duplicates as well, but it stores the original (possibly relative) path, and I decided it would be best to leave it alone.

Both are "loaded", so I might just call the second variable `configFilesConanical`